app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/home/HomeViewModel.kt

```diff
@@ -20,8 +22,28 @@ class HomeViewModel : BaseViewModel() {
     // Setup
     val user = repo.user.groundState(null)
     val repos = repo.repos.groundState(emptyList())
+    val pullRequests = repo.pullRequests.groundState(emptyList())
 
     // UI
+
+    val userPullRequestState: Flow<List<PullRequestItemState>> =
```

Comment:
@LuisRocketRangel NIT: Not a huge deal but you could probably use a UI model object here similar to the UserRepositoryUiModel being used below, with the idea being that you could just only add the PR data that you need to display in the card.